sdk/core/core-aborter/package.json

```diff
@@ -37,7 +37,7 @@
     "node": ">=6.0.0"
   },
   "dependencies": {
-    "@azure/ms-rest-js": "^1.2.6",
+    "@azure/ms-rest-js": "1.8.7",
```

Comment:
If I use "^1.8.7" here, and "1.8.7" in `common-versions.json` rush complains about version mismatch.